queries/bash/highlights.scm

```diff
@@ -95,16 +101,12 @@
   descriptor: (file_descriptor) @operator
   destination: (word) @parameter)
 
-
-("$" (variable_name)) @identifier
-
 (expansion
   [ "${" "}" ] @punctuation.bracket)
 
-(variable_name) @identifier
+(variable_name) @variable
```

Comment:
I prefer having colored variables in shell scripts, they stand out from the rest of the code.
Maybe `@type` or `@variable.builtin` would do better ? `@variable` should be like Normal in most cases.

Reply:
I think variable is fine here. The color can be set using the TSVariable highlight group.

Reply:
I don't agree. if you'd set TSVariable to a color, then it would apply for every language, and i would not like my variables to be colored in other languages. Imagine every variable is colored in c or rust. This would make the code unreadable.
Basically, only viml and shell imo should be highlighted as such.
If you think it should stay `@variable`, i will override this in a query of mine, no worries :)

Reply:
I was feeling the same about `@variable`, but then it will be kind of inconsistent with the rest of languages. Not sure if hl groups can be overridden by filetype

Reply:
not sure, but i would prefer overwriting queries. I might even extend my colorscheme with custom queries in this case

Reply:
word is being overridden later if it matches a number/boolean/constant, so I think we are good to assume it's a string if it doesn't match the other ones.